pkg/config/deployment.go

```diff
@@ -29,6 +29,7 @@ const (
 )
 
 type GenericDeploymentSpec struct {
+	model.ApplicationInfo `json:",inline"`
```

Comment:
This will also include unnecessary files such as `kind` `config_filename`. So maybe we should define them explicitly here.

Reply:
for sure